source/free-monitoring.txt

```diff
@@ -0,0 +1,68 @@
+.. _free_monitoring-mongosh:
+
+===============
+Free Monitoring
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. versionadded:: 1.4.1
+
+When connecting to MongoDB Community Edition clusters
+with ``mongosh`` you will be prompted with instructions on enabling 
+MongoDB's free monitoring service.
+
+To enable free monitoring, run the following command: 
+
+.. code-block:: sh
+
+   db.enableFreeMonitoring()
+
+To permanently disable this reminder, run the following command: 
+
+.. code-block:: sh
+
+   db.disableFreeMonitoring()
+
+Monitored Data
+--------------
+
+Free monitoring provides information about your deployment, including:
+
+- Operation Execution Times
+
+- Memory Usage
+
+- CPU Usage
+
+- Operation Counts
+
+The data expires 24 hours after being uploaded.
+
+Monitored Data and Expiration
+-----------------------------
+
+When enabled, the monitored data is uploaded periodically. The
+monitored data expires after 24 hours. That is, you can only access
+monitored data that has been uploaded within the past 24 hours.
+
+If you disable free monitoring and later re-enable free monitoring, you
+can access your previous metrics that have not expired within the past
```

Comment:
Consider:

If you disable, and then re-enable, monitoring, you can still access metrics that have not expired within the past 24 hours.

(or "Free Monitoring" if it's a product name)